### PR TITLE
Enable Prettier check over instrumentation/python

### PR DESCRIPTION
**Pull request by chalin**
- Contributes to #1063
- Enables format checking over `instrumentation/python` section
- Replaces all `relref` links by standard markdown links

Preview: https://deploy-preview-2338--opentelemetry.netlify.app/docs/instrumentation/python/

### CODE REVIEW

File: content/en/docs/instrumentation/python/automatic/example.md [hunk 1]
```diff
@@ -273,31 +276,30 @@ If those headers are available, they will be included in your span:
...
-        ]
-    }
+  "attributes": {
+    "http.request.header.user-agent": [
+      "Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 5.1; Trident/4.0)"
+    ],
+    "http.request.header.accept_encoding": ["gzip, deflate, br"],
+    "http.response.header.last_modified": ["2022-04-20 17:07:13.075765"],
+    "http.response.header.content_type": ["text/html; charset=utf-8"]
+  }
```
Comment (chalin) [inline content/en/docs/instrumentation/python/automatic/example.md:286]: Note the reformatting of the JSON code block. /cc @cartermp et al.

**Comment by chalin** [comment]
Note to reviewers: Prettier reformats a few JSON and YAML code blocks. IMHO, that's ok, but I wanted to point that out.

File: content/en/docs/instrumentation/python/automatic/example.md (hunk 1 above)
Comment (svrnm) [reply content/en/docs/instrumentation/python/automatic/example.md:286]: I am OK with that.

**Comment by chalin** [comment]
Given that this is just a reformatting PR, is this good to merge @svrnm et al.?

**Review by cartermp — APPROVED** [review]
merge'ums

**Pull request merged**